include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -65,68 +65,67 @@ namespace seqan3
  */
 /*!\name Requirements for seqan3::aligned_sequence_concept
  * \brief You can expect these functions on all types that model the seqan3::aligned_sequence_concept.
+ * \relates seqan3::aligned_sequence_concept
  *
  * \details
  *
  * Type Requirements:
- *   1. The type must model the std::ranges::ForwardRange.
- *   2. The `value_type` must model the alphabet concept.
- *   3. The `value_type` must be assignable from seqan3::gap::GAP.
+ *   1. The type must model std::ranges::ForwardRange.
+ *   2. The `reference type` must model seqan3::alphabet_concept.
+ *   3. The `reference type` must be assignable from seqan3::gap.
  *
  * Furthermore, the free functions seqan3::aligned_sequence_concept::insert_gap
  * and seqan3::aligned_sequence_concept::erase_gap must exist for the type.
```

Comment:
but you don't need to mention all the constraints again in the group description, they are already described in the concept, or not?